src/cloudwatch-applicationsignals-mcp-server/evals/applicationsignals/enablement_tasks.py

```diff
@@ -0,0 +1,232 @@
+
+
+class EnablementTask(Task):
+    """Task for evaluating Application Signals enablement.
+
+    Tests whether the agent can:
+    1. Call get_enablement_guide MCP tool correctly
+    2. Understand the returned enablement instructions
+    3. Modify IaC and application files appropriately
+    4. Pass build validation and rubric criteria
+    """
+
+    def __init__(
+        self,
+        id: str,
+        prompt_template: str,
+        git_paths: list[str],
+        iac_dir: str,
+        app_dir: str,
+        language: str,
+        framework: str,
+        platform: str,
+        validation_rubric: list[str],
+        expected_tools: Optional[list[str]] = None,
+        build_command: Optional[str] = None,
+        build_working_dir: Optional[str] = None,
+        modifies_code: bool = True,
+    ):
+        """Initialize EnablementTask.
+
+        Args:
+            id: Task identifier
+            prompt_template: Prompt passed to the AI agent doing Application Signals enablement
+            git_paths: List of paths (relative to working_directory) for git diff/cleanup
+            iac_dir: IaC directory path (relative to working_directory)
+            app_dir: Application directory path (relative to working_directory)
+            language: Programming language (e.g., 'python', 'java')
+            framework: Framework (e.g., 'flask', 'spring-boot')
+            platform: Platform (e.g., 'ec2', 'ecs', 'eks')
+            validation_rubric: List of validation criteria
+            expected_tools: Expected MCP tools to be called
+            build_command: Optional build command (e.g., 'npm install && npm run build')
+            build_working_dir: Optional build working directory (relative to working_directory)
+            modifies_code: Whether task modifies files (for cleanup)
+        """
+        super().__init__(id=id)
+        self.prompt_template = prompt_template
+        self.git_paths = git_paths
+        self.iac_dir = iac_dir
+        self.app_dir = app_dir
+        self.language = language
+        self.framework = framework
+        self.platform = platform
+        self.validation_rubric = validation_rubric
+        self.expected_tools = expected_tools or ['get_enablement_guide']
+        self.build_command = build_command
+        self.build_working_dir = build_working_dir
+        self.modifies_code = modifies_code
+
+    def get_working_directory(self):
+        """Return path to Application Signals samples directory.
+
+        Returns:
+            Path to cloudwatch-applicationsignals-mcp-server samples directory
+        """
+        # Calculate path to samples: enablement_tasks.py -> applicationsignals/ -> evals/
+        # -> cloudwatch-applicationsignals-mcp-server/ -> src/ -> mcp/ -> samples/
+        return (
+            Path(__file__).parent.parent.parent.parent.parent
+            / 'samples'
+            / 'cloudwatch-applicationsignals-mcp-server'
+        )
```

Comment:
```suggestion
        return (
            Path(__file__).parent.parent.parent.parent.parent
            / 'samples'
            / 'cloudwatch-applicationsignals-mcp-server/get-enablement-guide-samples/'
        )
```

Though IMO we should create a temp dir and copy-paste all relevant files there in setup and delete files in cleanup.